kiwi/repository/dnf.py

```diff
@@ -80,7 +80,8 @@ def post_init(self, custom_args=None):
         self.shared_dnf_dir = {
             'reposd-dir': manager_base + '/repos',
             'cache-dir': manager_base + '/cache',
-            'pluginconf-dir': manager_base + '/pluginconf'
+            'pluginconf-dir': manager_base + '/pluginconf',
+            'vars-dir': manager_base + '/vars'
```

Comment:
We should add `varsdir` setting to the generated `dnf.conf` file, as that can be used by DNF too, and I could see reasons for us to want to populate that in the future.

Reply:
ok will adapt regarding this

Reply:
done